src/pipeline/container.go

```diff
@@ -89,3 +89,23 @@ func waitContainer(id string) (int, error) {
 
 	return client.WaitContainer(id)
 }
+
+func isImageLocal(image string) bool {
+	repository := strings.Split(image, ":")
```

Comment:
There is a function at https://github.com/fsouza/go-dockerclient/blob/master/misc.go#L50 that does this can you use that?